README.md

```diff
@@ -91,6 +91,77 @@ The **PAW Workflow Extension** automates work item initialization, eliminating m
 
 The extension streamlines initialization but is not required—you can create the directory structure manually following the [PAW Specification](paw-specification.md).
 
+## Workflow Modes
```

Comment:
Let's make this section in the README more terse as not to take up too much space in the README. Instead the paw-specification.md should describe the details about the workflow modes and review strategies.